### PR TITLE
KAFKA-3256: Add print.timestamp option to console consumer.

### PR DESCRIPTION
**Pull request by becketqin**

### CODE REVIEW

**Comment by ijuma** [comment]
cc @junrao

File: core/src/main/scala/kafka/tools/ConsoleConsumer.scala [hunk 1]
```diff
@@ -358,8 +361,11 @@ class DefaultMessageFormatter extends MessageFormatter {
   }
 
   def writeTo(key: Array[Byte], value: Array[Byte], timestamp: Long, timestampType: TimestampType, output: PrintStream) {
-    if (timestampType != TimestampType.NO_TIMESTAMP_TYPE) {
-      output.write(s"$timestampType:$timestamp".getBytes)
+    if (printTimestamp) {
+      if (timestampType != TimestampType.NO_TIMESTAMP_TYPE)
+        output.write(s"$timestampType:$timestamp".getBytes)
+      else
+        output.write(s"NO_TIMESTAMP".getBytes)
```
Comment (junrao) [inline core/src/main/scala/kafka/tools/ConsoleConsumer.scala:368]: Should we write out the separator after the timestamp? Maybe we can just piggyback on keySeparator.

Comment (becketqin) [inline core/src/main/scala/kafka/tools/ConsoleConsumer.scala:368]: @junrao The key separator is written in line 369. Did you mean something else?

Comment (junrao) [inline core/src/main/scala/kafka/tools/ConsoleConsumer.scala:368]: Sorry, missed that.

**Comment by junrao** [comment]
Thanks for the patch. LGTM.

**Comment by becketqin** [comment]
Thanks @junrao

**Pull request closed without merging**